translatorbot/LICENSE

```diff
@@ -0,0 +1,25 @@
+BSD 2-Clause License
```

Comment:
I think we can have a single license file in the root, no need to include it again here.